terraform/modules/benchmark/service/run_benchmark.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+## Enable strict error handling
+set -eux
+
+# 1. Initialize s3cmd
+# 2. Run benchmark
+# 3. Upload benchmark results to s3 bucket
+
+## Configure s3cmd
+CFG=~/.s3cfg
+
+if [ ! -f "$CFG" ]; then
+    s3cmd --dump-config \
+        --host="$BENCHMARK_ENDPOINT" \
+        --host-bucket="%(bucket)s.$BENCHMARK_ENDPOINT" \
+        --access_key="$AWS_ACCESS_KEY_ID" \
+        --secret_key="$AWS_SECRET_ACCESS_KEY" \
+        --multipart-chunk-size-mb=4096 > "$CFG"
+    echo "Configuration file created at $CFG"
+else
+    echo "s3cmd Configuration file exist at $CFG"
+fi
+
+## Run actual benchmark
+ruby bench.rb --chain calibnet --tempdir ./tmp --daily
+ruby bench.rb --chain mainnet --tempdir ./tmp --daily
+
+## Upload benchmark result to s3 weekly file
+week_number=$(date +%W) # Week starting on Monday
+s3cmd get s3://"$BENCHMARK_BUCKET"/benchmark-results/weekly-results/weekly_result_"$week_number".csv /tmp/weekly_result_"$week_number".csv --force || 
+echo "Timestamp,Forest Version,Lotus Version,Chain,Metric,Forest Value,Lotus Value" > /tmp/weekly_result_"$week_number".csv
+tail -n +2 -q /chainsafe/result_*.csv >> /tmp/weekly_result_"$week_number".csv && s3cmd --acl-public put /tmp/weekly_result_"$week_number".csv s3://"$BENCHMARK_BUCKET"/benchmark-results/weekly-results/weekly_result_"$week_number".csv || exit 1
+
+s3cmd get s3://"$BENCHMARK_BUCKET"/benchmark-results/all_results.csv /tmp/all_results.csv --force || 
+echo "Timestamp,Forest Version,Lotus Version,Chain,Metric,Forest Value,Lotus Value" > /tmp/all_results.csv
+tail -n +2 -q /chainsafe/result_*.csv >> /tmp/all_results.csv && s3cmd --acl-public put /tmp/all_results.csv s3://"$BENCHMARK_BUCKET"/benchmark-results/all_results.csv || exit 1
```

Comment:
Why do we need to `exit 1` in case of an error? Shouldn't `set -e` that is already in the file work fine?